CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## 15.4.16 (08/23/24)
+
```

Comment:
@fheinecke We should add a separate section on top here describing the vulnerability we're making public:

```
### Security fix

#### [High] Stored XSS in SAML IdP

When registering a service provider with SAML IdP, Teleport did not sufficiently validate the ACS endpoint. This
could allow a Teleport administrator with permissions to write saml_idp_service_provider resources to configure
a malicious service provider with an XSS payload and compromise session of users who would access that
service provider.

Note: This vulnerability is only applicable when Teleport itself is acting as the identity provider. If you only use
SAML to connect to an upstream identity provider you are not impacted. You can use the tctl get
saml_idp_service_provider command to verify if you have any Service Provider applications registered and
Teleport acts as an IdP.

For self-hosted Teleport customers that use Teleport as [SAML Identity Provider](https://goteleport.com/docs/access-controls/idps/saml-guide/),
we recommend upgrading auth and proxy servers. Teleport agents (SSH, Kubernetes, desktop, application,
database and discovery) are not impacted and do not need to be updated.

### Other fixes and improvements
```